src/app/app.routes.ts

```diff
@@ -1,3 +1,25 @@
 import { Routes } from '@angular/router';
+import {HomeComponent} from './home/home.component'
+
+
+export const routes: Routes = [
+  {
+    path: "GeneralChat",
```

Comment:
I believe the URL path should be structured differently, for example, as: `general-chat/math-chat`